Styleguide.md

```diff
@@ -190,17 +193,17 @@ When programmatically building up strings, use template strings instead of conca
 ```js
 // bad
 function sayHi(name) {
-  return 'How are you, ' + name + '?';
+  return "How are you, " + name + "?";
 }
 
 // bad
 function sayHi(name) {
-  return ['How are you, ', name, '?'].join();
+  return ["How are you, ", name, "?"].join();
 }
 
 // bad
 function sayHi(name) {
-  return `How are you, ${ name }?`;
+  return `How are you, ${name}?`;
```

Comment:
How is this different from the good example?

Reply:
removed